docs/api/remote-procedure-calls-wallet.md

```diff
@@ -965,8 +970,9 @@ _Result---a description of the transaction_
 | → →<br>`fee`                 | number (dash)   | Optional<br>(0 or 1)        | If sending payment, the fee paid as a negative dash value.  May be `0`.  Not returned if receiving payment                                                                                                                                                                                                                                                                                    |
 | → →<br>`abandoned`           | bool            | Optional<br>(0 or 1)        | _Added in Bitcoin Core 0.12.1_<br><br>Indicates if a transaction is was abandoned:<br>• `true` if it was abandoned (inputs are respendable)<br>• `false`  if it was not abandoned<br>Only returned by _send_ category payments                                                                                                                                                                |
 | →<br>`hex`                   | string (hex)    | Required<br>(exactly 1)     | The transaction in serialized transaction format                                                                                                                                                                                                                                                                                                                                              |
+| →<br>`decoded`               | object          | Optional<br>(0 or 1)        | the decoded transaction (only present when `verbose` is passed), equivalent to the`verbose` is passed), equivalent to the RPC decoderawtransaction method, or the RPC getrawtransaction method when `verbose` is passed.                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
Several typo/formatting issues here. Let's also make references to other RPCs links to them.